Author: "A. Costa" <agcosta@gis.net>
Author: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Author: Christoph Hellwig <hch@lst.de>
Bug-Debian: http://bugs.debian.org/598714
Bug-Debian: http://bugs.debian.org/575201
Reviewed-by: Rogério Theodoro de Brito <rbrito@ime.usp.br>
Last-Update: 2010-11-25
Forwarded: no
Description:
  Remove typos, references in the "See also" sections of the manpages to
  commands that are not available in Debian systems (and, quite probably,
  other systems too) and non-Linux specific comments.
--- a/newfs_hfs.tproj/newfs_hfs.8
+++ b/newfs_hfs.tproj/newfs_hfs.8
@@ -19,10 +19,10 @@
 .Dt NEWFS_HFS 8
 .Os "Mac OS X"
 .Sh NAME
-.Nm newfs_hfs
+.Nm mkfs.hfs
 .Nd construct a new HFS Plus file system
 .Sh SYNOPSIS
-.Nm newfs_hfs
+.Nm mkfs.hfs
 .Op Fl N Ar [partition-size]
 .Op Fl U Ar uid
 .Op Fl G Ar gid
@@ -37,19 +37,13 @@
 .Op Fl v Ar volume-name
 .Ar special
 .Sh DESCRIPTION
-.Nm Newfs_hfs
+.Nm mkfs.hfs
 builds an HFS Plus file system on the specified special device.
-Before running 
-.Nm newfs_hfs
-the disk should be partitioned using the
-.Em Disk Utility
-application or
-.Xr pdisk 8 .
 .Pp
 The file system default parameters are calculated based on
 the size of the disk partition. Typically the defaults are
 reasonable, however
-.Nm newfs_hfs
+.Nm mkfs.hfs
 has several options to allow the defaults to be selectively overridden.
 The options are as follows:
 .Bl -tag -width Fl
@@ -66,7 +60,7 @@
 Specify the access permissions mask for the file system's root directory.
 .It Fl h
 Creates a legacy HFS format filesystem. This option
-is not recomended for file systems that will be
+is not recommended for file systems that will be
 primarily used with Mac OS X or Darwin.
 .It Fl s
 Creates a case-sensitive HFS Plus filesystem. By
@@ -139,8 +133,7 @@
 Volume name (file system name) in ascii or UTF-8 format.
 .El
 .Sh SEE ALSO
-.Xr mount 8 ,
-.Xr pdisk 8
+.Xr mount 8
 .Sh HISTORY
 The
 .Nm
--- a/fsck_hfs.tproj/fsck_hfs.8
+++ b/fsck_hfs.tproj/fsck_hfs.8
@@ -19,18 +19,18 @@
 .Dt FSCK_HFS 8
 .Os "Mac OS X"
 .Sh NAME
-.Nm fsck_hfs
+.Nm fsck.hfs
 .Nd HFS file system consistency check
 .Sh SYNOPSIS
-.Nm fsck_hfs
+.Nm fsck.hfs
 .Fl q
 .Op Fl df
 .Ar special ...
-.Nm fsck_hfs
+.Nm fsck.hfs
 .Fl p
 .Op Fl df
 .Ar special ...
-.Nm fsck_hfs
+.Nm fsck.hfs
 .Op Fl n | y | r
 .Op Fl dfgl
 .Op Fl m Ar mode
@@ -52,9 +52,7 @@
 preens the specified file systems.
 It is normally started by
 .Xr fsck 8
-run from
-.Pa /etc/rc.boot
-during automatic reboot, when a HFS file system is detected.
+during systen boot, when a HFS file system is detected.
 When preening file systems,
 .Nm
 will fix common inconsistencies for file systems that were not
@@ -105,9 +103,9 @@
 .It Fl g
 Causes
 .Nm
-to generate it's output strings in GUI format.
+to generate its output strings in GUI format.
 This option is used when another application with a graphical user interface
-(like Mac OS X Disk Utility) is envoking the
+(like Mac OS X Disk Utility) is invoking the
 .Nm
 tool.
 .It Fl l
@@ -144,8 +142,6 @@
 to the leaf nodes in the existing catalog file.
 .El
 .Pp
-Because of inconsistencies between the block device and the buffer cache,
-the raw device should always be used.
 .Sh SEE ALSO
 .Xr fsck 8
 .Sh BUGS
